src/Estimators/CSEnergyEstimator.cpp

```diff
@@ -14,7 +14,7 @@
 //////////////////////////////////////////////////////////////////////////////////////
 
 
-#include "Estimators/CSEnergyEstimator.h"
+#include "CSEnergyEstimator.h"
```

Comment:
"Estimators/CSEnergyEstimator.h"

Reply:
This is on purpose.
CSEnergyEstimator.cpp should include CSEnergyEstimator.h without any directory if they are in the same directory.

Reply:
https://qmcpack.readthedocs.io/en/develop/developing.html#includes
That's not our published coding standard.
All my new code explicitly qualifies internal include location under src

Reply:
I believe "" are looked for in the entire non "system" include path set, this standard is usually thought to remove sloppy dependence on that.

Reply:
https://gcc.gnu.org/onlinedocs/cpp/Include-Syntax.html
"" looks for the directory of the cpp file first and guarantees matching h an cpp.
I think we need to update the coding standard.

Reply:
I am now in agreement especially re goal of cmake targets determining include paths.